spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -141,4 +141,48 @@ class PowerShellManager; end
     end
   end
 
+  describe "when output is written to a PowerShell Stream" do
```

Comment:
Do we want to test a script that uses merged output like `3>&2`?  (or anything that doesn't merge it to `1`)